tests/FINDOLOGIC/Export/Tests/Helper/Utility.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace FINDOLOGIC\Export\Tests\Helper;
+
+/**
+ * Class Utility
+ * @package FINDOLOGIC\Export\Tests\Helper
+ */
+class Utility
```

Comment:
Rather than introducing yet another utility class, I think it's better to rather introduce a shared base class for tests and put the helper method there if it is used in a large number of tests classes. Alternatively, if it is used in multiple test classes, but not the majority of them, one could consider putting this into a trait used by the relevant test classes.

@mmachatschek What do you think?

Reply:
@howard I simply moved the method back to the test class because, we do not need this type of generator in other classes. We can consider introducing a TestBase class at another point I think.

If you have any other thoughts on this, bump me!